modules/op-disabling-cluster-tasks-and-pipeline-templates.adoc

```diff
@@ -0,0 +1,33 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/customizing-configurations-in-the-tektonconfig-cr.adoc
+
+:_content-type: CONCEPT
+[id="op-disabling-cluster-tasks-and-pipeline-templates_{context}"]
+= Disabling cluster tasks and pipeline templates
+
+[role="_abstract"]
+By default, the `TektonAddon` CR installs `clusterTasks` and `pipelineTemplates` resources along with {pipelines-shortname} on the cluster.
+
+You can disable the installation of the `clusterTasks` and `pipelineTemplates` resources by changing the values under `.spec.addon.params` to `false`. In addition, you can disable `communityClusterTasks` after disabling `clusterTasks`.
```

Comment:
> In addition, you can disable `communityClusterTasks` after disabling `clusterTasks`.

@VeereshAradhya this doesn't sound correct, does it? IMHO you can disable community cluster tasks or *all* cluster tasks, right?

Reply:
My bad. You are right, it is actually the other way around: we need to disable community cluster tasks before disabling cluster tasks. I accidentally put it the other way around. It is corrected now.